### PR TITLE
feat(ONYX-1015): add Artwork Title step

### PR DESCRIPTION
**Pull request by tam-kis**
The type of this PR is: Feat

This PR solves [ONYX-1015](https://artsyproduct.atlassian.net/browse/ONYX-1015)

### Description

Add Artwork Title step in new submission flow.

https://github.com/artsy/force/assets/17580625/8e58a7f9-3c93-4687-83cf-e4d668fc14c2

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ONYX-1015]: https://artsyproduct.atlassian.net/browse/ONYX-1015?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by olerichter00 — DISMISSED** [review]
🌟

File: src/Apps/Sell/Routes/TitleRoute.tsx [hunk 1]
```diff
@@ -48,16 +62,39 @@ export const TitleRoute: React.FC<TitleRouteProps> = props => {
...
+                    {submission?.artist?.formattedNationalityAndBirthday}
+                  </Text>
+                </Flex>
+              </Flex>
+            </Flex>
+            <Input
+              onChange={handleChange}
+              name="title"
+              placeholder="Artwork Title"
+              defaultValue={values.title} // This is your GraphQL fragment
```
[the remark this answers is not in the document]
Comment (tam-kis) [reply src/Apps/Sell/Routes/TitleRoute.tsx]: This was leftover from the original scaffolding, I guess it's ok to remove it now.

File: src/Apps/Sell/Routes/__tests__/TitleRoute.jest.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { useRouter } from "System/Router/useRouter"
+import { graphql } from "react-relay"
+import { SubmissionRoute } from "Apps/Sell/Routes/SubmissionRoute"
+
+const mockUseRouter = useRouter as jest.Mock
+const mockPush = jest.fn()
+const mockReplace = jest.fn()
+
+jest.mock("System/Router/useRouter", () => ({
```
Comment (damassi) [inline src/Apps/Sell/Routes/__tests__/TitleRoute.jest.tsx:13]: Going to need to rebase

Comment (tam-kis) [reply src/Apps/Sell/Routes/__tests__/TitleRoute.jest.tsx:13]: Yep, saw your message, thanks for letting us know!

**Review by olerichter00 — APPROVED** [review]

**Pull request merged**